### PR TITLE
Solution 06 time management module

### PR DESCRIPTION
**Pull request by pahomov-and**

### CODE REVIEW

File: 06-TimeManagement/module/module_06.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		kobject_put(kobject_module_06);
+
+	pr_info("%s: module start: %d\n",  __func__, retval);
+	return retval;
+}
+
+static void module_06_exit(void)
+{
+	pr_info("%s: module exit\n",  __func__);
+	kobject_put(kobject_module_06);
```
Comment (yekovalyov) [inline 06-TimeManagement/module/module_06.c:203]: Is this call of kobject_put() correct?

**Comment by yekovalyov** [comment]
Please squash your commits in one for this PR.

File: 06-TimeManagement/module/module_06.c (hunk 1 above)
Comment (pahomov-and) [reply 06-TimeManagement/module/module_06.c:203]: It works correctly but you're right I need to add sysfs_remove_group

File: 06-TimeManagement/module/module_06.c [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+	a = (time_idle - old_idle)*10000;
+	b = time_system - old_system;
+	c = 10000 - div64_u64(a, b);
+
+	real = div64_u64_rem(c, 100, &rem);
+
+	cpu_procent.H = real;
+	cpu_procent.L = rem;
+
+	pr_info("CPU: %llu.%llu %%\n",  cpu_procent.H, cpu_procent.L);
```
Comment (yekovalyov) [inline 06-TimeManagement/module/module_06.c]: Is the second parameter correct?

File: 06-TimeManagement/module/module_06.c (hunk 1 above)
Comment (yekovalyov) [reply 06-TimeManagement/module/module_06.c:203]: What does is mean 'work correctly'? Did not generate segfault.

File: 06-TimeManagement/module/module_06.c [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+	return idle;
+}
+
+
+static ssize_t real_loadavg_show(struct kobject *kobj,
+	struct kobj_attribute *attr, char *buf)
+{
+
+	return sprintf(buf, "CPU: %llu.%llu %%\n",
```
Comment (yekovalyov) [inline 06-TimeManagement/module/module_06.c]: Error in format string.

File: 06-TimeManagement/module/module_06.c [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+	static u64 ticks_old;
+	u64 rem;
+
+	ticks_now = get_jiffies_64();
+
+	ticks_temp = ticks_now - ticks_old;
+	ticks_temp = div64_u64_rem(ticks_temp, HZ, &rem);
+
+	ticks_old = ticks_now;
+	return sprintf(buf, "dt: %llu.%llu s\n", ticks_temp, rem);
```
Comment (yekovalyov) [inline 06-TimeManagement/module/module_06.c]: Error in format string.

File: 06-TimeManagement/module/module_06.c (hunk 2 above)
Comment (yekovalyov) [inline 06-TimeManagement/module/module_06.c]: Wrong format line.

**Pull request merged**